telegram/_user.py

```diff
@@ -98,7 +105,14 @@ class User(TelegramObject):
         supports_inline_queries (:obj:`str`): Optional. :obj:`True`, if the bot supports inline
             queries. Returned only in :attr:`telegram.Bot.get_me` requests.
         bot (:class:`telegram.Bot`): Optional. The Bot to use for instance methods.
+        is_premium (:obj:`bool`, optional): Optional. :obj:`True`, if this user is a Telegram
+            Premium user.
+
+            .. versionadded:: 20.0
+        added_to_attachment_menu (:obj:`bool`, optional): Optional. :obj:`True`, if this user added
+            the bot to the attachment menu.
```

Comment:
```suggestion
        is_premium (:obj:`bool`): Optional. :obj:`True`, if this user is a Telegram
            Premium user.

            .. versionadded:: 20.0
        added_to_attachment_menu (:obj:`bool`): Optional. :obj:`True`, if this user added
            the bot to the attachment menu.
```

Reply:
nice catches

Reply:
ty, I saw it because I went through them again for V13